src/modules/Providers/ProvidersWrapper.tsx

```diff
@@ -1,7 +1,9 @@
+import * as React from 'react';
 import withQueryClient from 'src/components/QueryClientHoc';
 
 import ProvidersPage from './ProvidersPage';
 
-const Page = withQueryClient(ProvidersPage);
+const ProvidersWrapper: React.FC = withQueryClient(ProvidersPage);
+ProvidersWrapper.displayName = 'ProvidersWrapper';
```

Comment:
how about adding the `displayName` to ProvidersPage? This way the HoC should create the name for us.

Reply:
As is currently: 
![image](https://user-images.githubusercontent.com/3985964/211637971-4a6a7e10-d6ea-47ca-a82b-6c04cf63d946.png)

If I comment out this displayName:
![image](https://user-images.githubusercontent.com/3985964/211638403-b4ade0b8-e66c-4e82-8c90-4a011461b178.png)

So as is shows the component as **ProvidersWrapper** and letting `withQueryClient()` decide it is **ProvidersPage QueryClientHoc**.

I'm 50/50 on which way to go.

Reply:
Actually, we can side step some of this in prod builds by using `keep_fnames` as a webpack optimization `TerserPlugin` config.  It is the minimization step in prod mode that by default would throw out function names.

For example in ovirt-web-ui: https://github.com/oVirt/ovirt-web-ui/blob/dfe0c4b8c92638f6e41b9fe0b09e0d07509618ae/config/webpack.config.prod.js#L73-L79